src/utils/responsive-props.ts

```diff
@@ -43,3 +44,37 @@ export const responsiveSpacing = (
 
   return style
 }
+
+export const responsivePadding = (
+  customPropertyName: string,
+  propValues?: ResponsivePropValues,
+) => {
+  // Type guard to check if PropValue is PaddingObject
+  const instanceOfPaddingObject = (value: PropValue): value is IPaddingObject => {
+    return (value as IPaddingObject).top !== undefined
+  }
+  const style: { [index: string]: any } = {}
+  const breakpoints: String[] = getBreakpointKeys()
+
+  convertToArray(propValues).forEach((value, index) => {
+    const breakpoint = breakpoints[index]
+
```

Comment:
super trivial comment: could deconstruct value object here to make it slightly neater later.  Clearly, I couldn't find anything to change in this PR - looks great 😎


```suggestion
const { top, right, bottom, left } = value as IPaddingObject
```

Reply:
Great thanks, I'll make a note 👍